Doctrine/Orm/OrderFilter.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Dunglas\ApiBundle\Doctrine\Orm;
+
+use Doctrine\ORM\QueryBuilder;
+use Dunglas\ApiBundle\Api\ResourceInterface;
+use Dunglas\ApiBundle\Doctrine\Orm\FilterInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\VarDumper\VarDumper;
+
+/**
+ * {@inheritdoc}
+ *
+ * @author Théo FIDRY <theo.fidry@gmail.com>
+ */
+class OrderFilter implements FilterInterface
+{
+    /**
+     * @var array List of properties by witch the collection can or cannot be ordered.
+     */
+    private $sourceList = [];
+
+    /**
+     * @var bool If true $this#sourcelist is used as a whitelist. Otherwise is used as a blacklist.
+     */
+    private $type = true;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return 'order';
+    }
+
+    /**
+     * @param array   $properties List of property names on which the filter will be enabled/disabled.
+     * @param boolean $type       If true properties are passed as a whitelist and as a blacklist otherwise.
+     */
+    public function __construct(array $properties = [], $type = true)
+    {
+        $this->sourceList = $properties;
+        $this->type       = $type;
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * Order collection by properties. The order of the ordered properties is the same as the order specified in the
+     * query.
+     */
+    public function apply(ResourceInterface $resource, QueryBuilder $queryBuilder, Request $request)
+    {
+        $this->applyFilter($resource, $queryBuilder, $request->query->get('order'));
```

Comment:
Can you make the property name configurable the same way it's done for `page` and `itemsByPage`?